source/reference/program/mongoexport.txt

```diff
@@ -192,7 +201,7 @@ shell uses the :doc:`mongoShell mode representation
 .. code-block:: javascript
 
    use test
-   db.traffic.insert( { _id: 1, volume: NumberLong(2980000), date: new Date() } )
+   db.traffic.insert( { _id: 1, volume: NumberLong("2980000"), date: new Date() } )
```

Comment:
Au contraire. Apparently if you don't include the quotes, it interprets it at a float, and sadness ensues. See: DOCS-4568.